src/framework/ui/checkbox/checkbox.component.tsx

```diff
@@ -46,48 +55,53 @@ export class CheckBox extends React.Component<Props> {
         borderWidth: style.borderWidth,
         borderColor: style.borderColor,
         backgroundColor: style.backgroundColor,
+        ...strictStyles.border,
       },
       icon: {
         width: style.size / 2,
         height: style.size / 8,
         backgroundColor: style.selectColor,
+        ...strictStyles.icon,
       },
       highlight: {
         width: style.highlightSize,
         height: style.highlightSize,
         borderRadius: style.highlightBorderRadius,
         backgroundColor: style.highlightColor,
         opacity: style.highlightOpacity,
+        ...strictStyles.highlight,
       },
     };
   };
 
-  public render(): React.ReactNode {
-    const componentStyle: StyleType = this.getComponentStyle(this.props.themedStyle);
+  public render(): React.ReactElement<TouchableOpacityProps> {
+    const { themedStyle, ...derivedProps } = this.props;
```

Comment:
In this case usually better work `rest`, `other`

Reply:
Agree with rest, but if we're going to do this it's better to make it everywhere. Can be planned